interfaces/vrep/DQ_SerialVrepRobot.m

```diff
@@ -0,0 +1,240 @@
+            %     
+            %     Note that this calls "set_joint_positions" in the remoteAPI, meaning that it
+            %     is only suitable for passive joints.
+
+            obj.vrep_interface.set_joint_positions(obj.joint_names, q)
+        end
+        
+        function q = get_configuration_space_positions(obj)
+            % This method gets the joint configurations of the robot in the CoppeliaSim scene.
+            % Usage:
+            %     get_configuration_space_positions;  
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     q = vrep_robot.get_configuration_space_positions;
+
+            q = obj.vrep_interface.get_joint_positions(obj.joint_names);
+        end
+
+        function set_target_configuration_space_positions(obj, q_target)
+            % This method sets the joint configurations to the robot in the CoppeliaSim scene as a target configuration for the joint controllers.
+            % Usage:
+            %     set_target_configuration_space_positions(q);  
+            %          q_target: The target joint configurations of the robot in the CoppeliaSim scene.
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     q_target = zeros(7,1);
+            %     vrep_robot.set_target_configuration_space_positions(q_target);
+            %     
+            %     Note that this calls "set_joint_target_positions" in the remoteAPI, meaning that it
+            %     is only suitable for active joints.
+
+            obj.vrep_interface.set_joint_target_positions(obj.joint_names, q_target)
+        end
+        
+        function qd = get_configuration_space_velocities(obj)
+            % This method gets the joint velocities of the robot in the CoppeliaSim scene.
+            % Usage:
+            %     get_configuration_space_velocities;  
+            %
+            % Example: 
+            %     vi = DQ_VrepInterface();
+            %     vi.connect('127.0.0.1',19997);
+            %     vrep_robot = DQ_SerialVrepRobot("my_robot", 7, "my_robot#1", vi);
+            %     qd = vrep_robot.get_configuration_space_velocities;
+            
+            qd = obj.vrep_interface.get_joint_velocities(obj.joint_names);
+        end
+
+        function set_target_configuration_space_velocities(obj, v_target)
+            % This method sets the joint velocities of the robot in the CoppeliaSim scene as a target velocity for the joint controllers.
+            % Usage:
+            %     set_target_configuration_space_positions(q);  
+            %          v_target: The target joint velocities of the robot in the CoppeliaSim scene.
```

Comment:
I'm a bit confused about the `v_target` placement. Shouldn't `v_target` be mentioned the usage?
This happens in other methods as well.

Reply:
It was copy and paste gore. Fixed in [e0f1a98](https://github.com/dqrobotics/matlab/pull/108/commits/e0f1a983b8813bc7a153e034f163b04f0c997ff5).